[CUBRIDMAN-266] [EN] 변경된 PL 서버 (cub_pl) 동작 변경에 대한 설명 추가/변경 (#624)
CUBRID/cubrid-manual: en/api/cciapi.rst

.. c:function:: int cci_register_out_param(int req_handle, int index)

>     The **cci_register_out_param** function is used to bind the parameters as outbind in Stored Procedure. The index value begins from 1.

Reviewer: ```suggestion
    The **cci_register_out_param** function is used to bind parameters as outbind in a stored procedure. The index value starts from 1.
```

Reviewer 2: 보통 명사라기보다는 고유 명사 (용어?) 같은 것이라 대문자가 더 어울리는 것 같습니다.

Reviewer: method/function/procedure와 마찬가지로, stored procedure도 일반적으로 쓰이는 용어(명사)로 오라클(https://docs.oracle.com/javase/tutorial/jdbc/basics/storedprocedures.html) 및 다른 매뉴얼에는 소문자로도 사용되는 것으로 보입니다.

현재 큐브리드 매뉴얼에는 stored procedure는 소문자로 사용되고 있어서 대문자로 바꾼다면 일관성을 위해 다 수정하는 작업이 필요할 듯 합니다.

        *    **CCI_ER_CON_HANDLE**
        *    **CCI_ER_USED_CONNECTION**
    

>     The following shows to print out "Hello, CUBRID" string with tored Procedure.

Reviewer: ```suggestion
    The following shows how to print the string "Hello, CUBRID" string using a stored procedure.
```

    
    ::
    
        $ vi cubrid.conf
    
        $ cubrid service start
        $ cubrid server start demodb
    

>     Implement and compile the class to be used as Stored Procedure.

Reviewer: ```suggestion
    Implement and compile the class to be used as a stored procedure.
```
